W-13934486 dev exper report bsj (#76)
mulesoft/docs-api-governance-documentation: modules/ROOT/pages/index.adoc

* *Improve your organization's API quality:* Identify conformance issues in governed APIs and take steps to resolve them.
* *Share and enforce governance best practices:* Customize and publish governance rulesets to share and enforce organization-specific best practices with your developers.

> * *Apply consistent rules from design time to deployment:* Apply governance rulesets from design time to deployment using centralized governance.

Reviewer: Are these bold bits UI elements? If not, remove the bold and consider putting the sentence in a paragraph below (for scannability). (Items in bold aren't translated.)

Author: I prefer this format for the intro, but I think Diane also told me she wants them in the two line format. This isn't new formatting, but I'll change it.

Reviewer: I'm mainly concerned about the bold being used for other than UI elements.

Author: Done

* *Enforce governance within your DevOps organization:* Automate API governance using CLI commands in your CI/CD pipeline.

== Anypoint API Governance Overview Video 

Watch the Anypoint API Governance Product Spotlight video to see a quick overview of Anypoint API Governance.

++++
include::_partials/api-gov-gs-video.html[]
++++

== Governance Console

> In the API Governance console, governance administrators can add <<gov-rulesets,governance rulesets>> to <<gov-profiles,governance profiles>> to apply the governance rulesets to multiple APIs across your organization. The API Governance console then provides you with an overview of conformance for all validated APIs.

Reviewer: Should "your" be "their" in this case? Or maybe something like "as a governance administrator, you can add..."

Author: Yes, this does switch voice. Again, it's picking between various evils. More wordy, 2nd voice, but having a role indicated.

Reviewer: You could change the "your" to "the" to avoid the voice switch. And remove the "you with" in the second sentence.

Author: I reworded slightly. Hopefully it helps. The new role makes it a little more awkward to talk about.

> Monitor your API conformance and notify developers to help improve the conformance.

Reviewer: "notify developers when APIs are not in conformance"?

Author: The idea is that Governance Officers are improving the overall conformance of many developers (potentially). I prefer to stick with how it's worded now.

// Screenshot of the console with callouts

include::_partials/api-gov-console-scap-callouts.adoc[]

== Governance Across Anypoint Platform

> In addition to viewing API conformance information in the API Governance console, developers,  architects, and implementors can view governance conformance information and take action to fix issues in several places in Anypoint Platform. These include:

Reviewer: ```suggestion
In addition to viewing API conformance information in the API Governance console, developers, architects, and implementors can view governance conformance information and fix issues from Anypoint Platform:
```

Reviewer: (It sounds like they're fixing issues with Anypoint Platform.) Should "in" be "on" or "from"? Is there a better word for places here?

Reviewer: Is API conformance information the same as governance conformance information? If so, can this sentence be streamlined?

Author: Our product team has been pretty adamant about "taking action to fix", since it isn't so simple as just fixing something. It's a fine point, I suppose. I'll try to break up the sentence, as it is a bit long.

Author: I'll commit your change in the spirit of the confusion of "from" vs. "in", which I get. I'll then add back some of the text to make it clear that they can't just magically fix everything from Anypoint Platform.
